openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -1143,3 +1146,74 @@ def get(self, request):
         Return the JWKS.
         """
         return JsonResponse(self.lti_tool_config.get_jwks(), safe=False)
+
+
+@require_safe
+def component_version_asset(request, component_version_uuid, asset_path):
+    """
+    Serves static assets associated with particular Component versions.
+
+    Important notes:
+    * This is meant for Studio/authoring use ONLY. It requires read access to
```

Comment:
If this should only be used by CMS, it might be good to enforce that?

I would love if the library API URLs just weren't defined for LMS, but without taking on all of that work, you could just:
```python
try:
    ensure_cms()
except ImproperlyConfigured:
    raise Http404()
```

Reply:
It is enforced–the URLs are [only configured for Studio](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_libraries/apps.py#L24-L27) and don't work in LMS. (I just verified this by trying out the LMS version of this URL and got the expected 404 debug messaging.

Reply:
Oh right, content libraries is a "plugin" 😂

Sounds good, thanks for checking.